collectors/ClusterPropertiesCollector.py

```diff
@@ -0,0 +1,90 @@
+from BaseCollector import BaseCollector
+import os, time, json
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+
+class ClusterPropertiesCollector(BaseCollector):
+    def __init__(self):
+        self.wait_for_inventory_data()
+        self.property_yaml = YamlRead('collectors/property.yaml').run()
+        self.g = GaugeMetricFamily('vrops_cluster_properties', 'testtest',
+                              labels=['datacenter', 'vccluster', 'propkey'])
+        self.name = self.__class__.__name__
+        self.post_registered_collector(self.name, self.g.name)
+
+    def describe(self):
+        yield self.g
+
+    def collect(self):
+        if os.environ['DEBUG'] >= '1':
+            print('ClusterPropertiesCollector starts with collecting the metrics')
+
+        for target in self.get_clusters_by_target():
+            token = self.get_target_tokens()
+            token = token[target]
+
+            if not token:
+                print("skipping " + target + " in " + self.name + ", no token")
+
+            uuids = self.target_clusters[target]
+            if 'number_metrics' in self.property_yaml[self.name]:
```

Comment:
good idea

Reply:
figured it out when debugging some code with carl. if we don't have all metric types it would error then.

Reply:
We have to change it everywhere.